automated/linux/docker-prune/prune-lib.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+verify_app_present () {
+    app="$1"
+
+    if docker ps | grep -q "${app}"; then
+        echo "${app} container found"
+        return 0
+    else
+        echo "${app} container not found"
+        return 1
+    fi
+}
+
+check_image () {
+    image_name="$1"
+
+    if [[ "$(docker images -q "${image_name}" 2> /dev/null)" == "" ]] ; then
+        echo "Docker image '${image_name}' not found."
+        return 1
+    else
+        echo "Docker image '${image_name}' found."
+        return 0
+    fi
+}
+
+get_image_sha () {
+    ls /var/sota/reset-apps/apps/shellhttpd > "$(pwd)/sha.txt"
+}
+
+check_image_prune () {
+    runtime="5 minutes"
+    endtime=$(date -ud "$runtime" +%s)
+
+    while true; do
+        if [ "$(date -u +%s)" -ge "$endtime" ]; then
+            echo "Not found"
+            return 1
+        elif (journalctl --no-pager -u aktualizr-lite | grep "Pruning unused docker containers"); then
+            echo "Found"
+            return 0
+        fi
+        sleep 1
+   done
+}
+
+compare_sha () {
+    if diff "$(pwd)/sha.txt" <(ls /var/sota/reset-apps/apps/shellhttpd) > /dev/null; then
+        return 1
+    else
```

Comment:
assuming else isn't needed here, just return.

Reply:
Cool, I did not think to try this